app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">Onionshare</string>
```

Comment:
Nitpick: on the website it's OnionShare

Reply:
Ooops. Good catch. I changed that in all instances I could fine to get used to that capitalization.